bundles/org.openhab.binding.fineoffsetweatherstation/src/main/java/org/openhab/binding/fineoffsetweatherstation/internal/domain/MeasureType.java

```diff
@@ -0,0 +1,156 @@
+import static org.openhab.binding.fineoffsetweatherstation.internal.FineOffsetWeatherStationBindingConstants.CHANNEL_PM25;
+import static org.openhab.binding.fineoffsetweatherstation.internal.FineOffsetWeatherStationBindingConstants.CHANNEL_PRESSURE;
+import static org.openhab.binding.fineoffsetweatherstation.internal.FineOffsetWeatherStationBindingConstants.CHANNEL_RAIN;
+import static org.openhab.binding.fineoffsetweatherstation.internal.FineOffsetWeatherStationBindingConstants.CHANNEL_RAIN_RATE;
+import static org.openhab.binding.fineoffsetweatherstation.internal.FineOffsetWeatherStationBindingConstants.CHANNEL_TEMPERATURE;
+import static org.openhab.binding.fineoffsetweatherstation.internal.FineOffsetWeatherStationBindingConstants.CHANNEL_UV_RADIATION;
+import static org.openhab.binding.fineoffsetweatherstation.internal.FineOffsetWeatherStationBindingConstants.CHANNEL_WATER_LEAK_DETECTION;
+import static org.openhab.binding.fineoffsetweatherstation.internal.FineOffsetWeatherStationBindingConstants.CHANNEL_WIND_SPEED;
+import static org.openhab.binding.fineoffsetweatherstation.internal.Utils.toInt16;
+import static org.openhab.binding.fineoffsetweatherstation.internal.Utils.toUInt16;
+import static org.openhab.binding.fineoffsetweatherstation.internal.Utils.toUInt32;
+import static org.openhab.binding.fineoffsetweatherstation.internal.Utils.toUInt8;
+import static org.openhab.core.library.unit.SIUnits.CELSIUS;
+import static org.openhab.core.library.unit.SIUnits.METRE;
+import static org.openhab.core.library.unit.SIUnits.PASCAL;
+import static org.openhab.core.library.unit.Units.DEGREE_ANGLE;
+import static org.openhab.core.library.unit.Units.METRE_PER_SECOND;
+import static org.openhab.core.library.unit.Units.MICROGRAM_PER_CUBICMETRE;
+import static org.openhab.core.library.unit.Units.MILLIMETRE_PER_HOUR;
+import static org.openhab.core.library.unit.Units.PARTS_PER_MILLION;
+import static org.openhab.core.library.unit.Units.PERCENT;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.function.BiFunction;
+
+import javax.measure.Unit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.fineoffsetweatherstation.internal.Utils;
+import org.openhab.core.library.types.DateTimeType;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.library.unit.Units;
+import org.openhab.core.thing.type.ChannelTypeUID;
+import org.openhab.core.types.State;
+
+/**
+ * Represents the measured type with conversion form the sensors' bytes to the openHAB state.
+ *
+ * @author Andreas Berger - Initial contribution
+ */
+@NonNullByDefault
+public enum MeasureType {
+
+    TEMPERATURE(CELSIUS, 2, CHANNEL_TEMPERATURE, (data, offset) -> toInt16(data, offset) / 10.),
```

Comment:
Is it possible to distinguish between indoor and outdoor temperatures? In this case system channel types could be used.

Reply:
You can override the channel Type at the `Measurand`, which I already did for the outdoor temperature. I'll adjust the channel type for the indoor temperature as well.